config/tests/src/test/java/org/jboss/windup/config/iteration/payload/IterationPayLoadPassTest.java

```diff
@@ -157,11 +163,17 @@ public void perform(GraphRewrite event, EvaluationContext context,
                         );
             return configuration;
         }
+        // @formatter:on
 
     }
 
-    public class TestIterationPayLoadNotPassProvider extends WindupRuleProvider
+    public class TestIterationPayLoadNotPassProvider extends AbstractRuleProvider
     {
+        public TestIterationPayLoadNotPassProvider()
+        {
+            super(MetadataBuilder.forProvider(TestIterationPayLoadNotPassProvider.class));
+        }
```

Comment:
I would rather see this metadata be given in getMetadata(Metadata m) function than in the constructor